assembly/onpremises-ide-packaging-war-platform-api/src/main/java/com/codenvy/api/deploy/OnPremisesIdeApiServletModule.java

```diff
@@ -48,14 +46,6 @@ protected void configureServlets() {
                 .through(com.codenvy.service.http.WorkspaceIdEnvironmentInitializationFilter.class);
         filterRegex("^/workspace/(?!config$|runtime$|.*:.*$).+")
                 .through(com.codenvy.service.http.WorkspaceIdEnvironmentInitializationFilter.class);
-        filterRegex("^/(account|creditcard)/(?!find|list).+").through(new AccountIdEnvironmentInitializationFilter(),
```

Comment:
also remove please account from mapping for LoginFilter here https://github.com/codenvy/codenvy/pull/181/files#diff-d7ca62cd2a04167dc47ecffc7497eb03R61 and maybe billing, creditcard and other like that